.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+
+[*.md]
+indent_size = 4
```

Comment:
Just a file to help with configuring whitespace for the project. Uses .editorconfig plugins, installed by default in Jetbrains IDE's but may need an extension in VSCode.